tests/unit_tests/olio/test_vdb.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+
+import resqpy.olio.vdb as vdb
+import numpy as np
+
+
+@pytest.mark.parametrize("byte_array, expected_array", [(b" \x01\x00\x00\xae\x01\x00\x00", np.array([288, 430])),
```

Comment:
I suggest replacing the leading space in the first byte array with the hexadecimal form, \x20 I believe.

Reply:
Added